src/Compilers/Extension/CompilerExtension.csproj

```diff
@@ -47,7 +47,6 @@
   <ItemGroup>
     <NuGetPackageToIncludeInVsix Include="Microsoft.DiaSymReader.Native" />
     <NuGetPackageToIncludeInVsix Include="Microsoft.Win32.Primitives" />
-    <!-- Even though the VS process includes both of these, we need to also include them in the VSIX for the compiler processes -->
```

Comment:
I agree the comment isn't accurate right now, but should it be kept in some form?